docker/README.md

```diff
@@ -0,0 +1,116 @@
+# Arcade AI Docker Compose Guide
+
+This guide provides detailed instructions on how to set up and run Arcade AI using Docker Compose.
+
+## Prerequisites
+
+-   **Docker** installed on your system. [Install Docker](https://docs.docker.com/get-docker/)
+-   **Docker Compose** installed. It comes bundled with Docker Desktop on Windows and macOS. For Linux, follow the [Docker Compose installation guide](https://docs.docker.com/compose/install/).
+
+## Getting Started
+
+### 1. Clone the Repository
+
+Begin by cloning the Arcade AI repository:
+
+```bash
+git clone https://github.com/ArcadeAI/arcade-ai.git
+```
+
+Change to the `docker` directory:
+
+```bash
+cd arcade-ai/docker
+```
+
+### 2. Copy and Configure Environment Variables
+
+Copy the example environment file to `.env`:
+
+```bash
+cp env.example .env
+```
+
+Open the `.env` file in your preferred text editor and fill in the required values. At a minimum, you **must** provide the `OPENAI_API_KEY`:
+
+```env:.env
+### LLM ###
+
+OPENAI_API_KEY=your_openai_api_key_here
+```
+
+If you plan to use other Large Language Model (LLM) providers, add their API keys as well:
+
+```env:.env
+ANTHROPIC_API_KEY=your_anthropic_api_key_here
+```
+
+_For a list of supported LLM providers, see the [Arcade AI documentation](https://docs.arcade-ai.com/integrations/models)._
+
+### 3. Run Docker Compose
+
+Start the Arcade AI services using Docker Compose:
+
+```bash
+docker compose -p arcade up
```

Comment:
@Spartee originally you had `docker compose up -p arcade` which did not work for me, so i updated to `docker compose -p arcade up`.

Also, I get the following error:
```
engine-1  | [GIN-debug] GET    /swagger/*any             --> github.com/swaggo/gin-swagger.CustomWrapHandler.func1 (4 handlers)
engine-1  | panic: runtime error: invalid memory address or nil pointer dereference
engine-1  | [signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0xe129e2]
engine-1  | 
engine-1  | goroutine 66 [running]:
engine-1  | github.com/ArcadeAI/Engine/internal/schedules.(*Manager).RunNextScheduledToolExecution(0xc000c4e030, {0x1621b20?, 0x282d460?})
engine-1  |     /build/internal/schedules/manager.go:101 +0xc2
engine-1  | github.com/ArcadeAI/Engine/internal.(*Engine).runBackgroundToolExecution.func1()
engine-1  |     /build/internal/engine.go:204 +0x86
engine-1  | created by github.com/ArcadeAI/Engine/internal.(*Engine).runBackgroundToolExecution in goroutine 14
engine-1  |     /build/internal/engine.go:201 +0x2f7
engine-1 exited with code 2
```